SU2_CFD/src/solvers/CSolver.cpp

```diff
@@ -2882,16 +2884,18 @@ void CSolver::Read_SU2_Restart_ASCII(CGeometry *geometry, const CConfig *config,
     }
   }
 
+  if (iPoint_Global != geometry->GetGlobal_nPointDomain())
+    SU2_MPI::Error("The solution file does not match the mesh.",CURRENT_FUNCTION);
```

Comment:
does not work for ASCII

Reply:
will you support it for this PR, or this is for later?

Reply:
It is a bit messy to do it, besides, if someone is restarting from ASCII the problem is probably not very large.
(every mpi rank needs to read the entire ASCII file because lines are defined by the \n character)